test/e2e/framework/kubelet_stats.go

```diff
@@ -602,15 +602,15 @@ func (r *resourceCollector) collectStats(oldStatsMap map[string]*kubeletstatsv1a
 func (r *resourceCollector) GetLatest() (e2ekubelet.ResourceUsagePerContainer, error) {
 	r.lock.RLock()
 	defer r.lock.RUnlock()
-	kubeletstatsv1alpha1 := make(e2ekubelet.ResourceUsagePerContainer)
+	kubeletstatsv1alpha1map := make(e2ekubelet.ResourceUsagePerContainer)
```

Comment:
`usagePerContainer`?
```suggestion
	usagePerContainer := make(e2ekubelet.ResourceUsagePerContainer)
```